packages/make-styles/config/api-extractor.local.json

```
@@ -0,0 +1,5 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/api-extractor/v7/api-extractor.schema.json",
+  "extends": "./api-extractor.json",
+  "mainEntryPointFilePath": "<projectFolder>/dist/<unscopedPackageName>/src/index.d.ts"
```

to fix your issue with api-extractor:

```suggestion
  "mainEntryPointFilePath": "<projectFolder>/dist/index.d.ts"
```